### PR TITLE
Workaround a11y issue with popups

### PR DESCRIPTION
**Pull request by Walingar**
## Proposed Changes

  - Add native initialization of ComposeAccessible that will create the proper mapping between ComposeAccessbile and ComposeAccessibleComponent on MacOs

See related PR in skiko: https://github.com/JetBrains/skiko/pull/793

## Testing

Test: a11y can be tested only manually for now

Reproducer:
1. Enable VoiceOver.
2. Open Menu with ctrl+option+space
3. Navigate to "click me" with Tab
4. Try to press it with ctrl+option+space

```kotlin
fun main() = singleWindowApplication {
    Column {
        var expanded by remember { mutableStateOf(false) }
        Column {
            Text(
                text = "Menu",
                modifier = Modifier.background(Color.Cyan)
                    .clickable(role = Role.Button, onClick = {
                        expanded = !expanded
                    })
            )
        }
        if (expanded) {
            Popup(
                popupPositionProvider = rememberComponentRectPositionProvider(),
                onDismissRequest = { expanded = false },
                focusable = true
            ) {
                Column {
                    Text(
                        text = "click me",
                        modifier = Modifier.clickable(
                            role = Role.Button,
                            onClick = { println("Click") })
                    )
                }
            }
        }
    }
}
```

Before the fix crash will occur:
```
Exception in thread “AWT-EventQueue-0” java.lang.IncompatibleClassChangeError: Class androidx.compose.ui.platform.ComposeSceneAccessible$ComposeSceneAccessibleContext does not implement the requested interface javax.accessibility.AccessibleAction
	at java.desktop/sun.lwawt.macosx.CAccessibility$3.run(CAccessibility.java:162)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:771)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:722)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:716)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:97)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:746)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:744)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:743)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

### CODE REVIEW

File: compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/platform/ComposeAccessible.kt [hunk 1]
```diff
@@ -78,8 +80,17 @@ internal class ComposeAccessible(
     var semanticsNode: SemanticsNode,
     val controller: AccessibilityControllerImpl? = null
 ) : Accessible {
+    private val isNativelyInitialized = AtomicBoolean(false)
+
     val accessibleContext: ComposeAccessibleComponent by lazy { ComposeAccessibleComponent() }
-    override fun getAccessibleContext(): AccessibleContext = accessibleContext
+
+    override fun getAccessibleContext(): AccessibleContext {
```
Comment (igordmn) [inline compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/platform/ComposeAccessible.kt:87]: > Test: a11y can be tested only manually for now

Could you add a reproducer of the issue to the description? So everyone can check and understand the fix in the future.

Comment (Walingar) [reply compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/platform/ComposeAccessible.kt:87]: Sure

Comment (igordmn) [reply compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/platform/ComposeAccessible.kt:87]: Maybe even add it in the comment itself

Comment (Walingar) [reply compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/platform/ComposeAccessible.kt:87]: Added to PR description, not sure that it is needed in the comment

Comment (igordmn) [reply compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/platform/ComposeAccessible.kt:87]: Checked it on macOs/Windows - it works (but didn't work on CM 1.4 macOS) 🎉. I had a crash in 1.4:
```
Exception in thread “AWT-EventQueue-0” java.lang.IncompatibleClassChangeError: Class androidx.compose.ui.platform.ComposeSceneAccessible$ComposeSceneAccessibleContext does not implement the requested interface javax.accessibility.AccessibleAction
	at java.desktop/sun.lwawt.macosx.CAccessibility$3.run(CAccessibility.java:162)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:771)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:722)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:716)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:97)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:746)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:744)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:743)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

Comment (igordmn) [reply compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/platform/ComposeAccessible.kt:87]: Worth to add it to the desc as well

**Review by igordmn — APPROVED** [review]

File: compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/platform/ComposeAccessible.kt (hunk 1 above)
Comment (Walingar) [reply compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/platform/ComposeAccessible.kt:87]: To desc of what?

**Pull request merged**